src/js/utils/styles.js

```diff
@@ -461,7 +461,9 @@ export const inputStyle = css`
         props.theme.global.input.font.weight};
     `} margin: 0;
   ${props => props.size && inputSizeStyle(props)}
-  ${props => props.focus && !props.plain && focusStyle()};
+  &:focus {
```

Comment:
Can we optimize the inclusion of `&:focus {}` within the conditional? For example:
```
${props => (!props.plain || props.focusIndicator) && `&:focus{ ${focusStyle()} }`}
```

Reply:
Sure thing!